### DISCUSSION TITLE
Is there any equivalent of (neo)vim's `:!` or `:terminal` in helix?

### DISCUSSION THREAD

**Discussion by zjp-CN** [helix-editor/helix#1765] [Q&A] [answered]
Helix is out of box to use rust-analyzer in Rust projects, and much smoother than (neo)vim.

But I couldn't find a way to execute `cargo build/run` or more general shell cmds in helix, except for [keymap#shell](https://docs.helix-editor.com/keymap.html#shell).

Maybe I've missed something. Much thanks if someone points out for me.

**Comment by kirawi** [comment, marked as answer]
https://github.com/helix-editor/helix/pull/1682
